.openpublishing.redirection.json

```diff
@@ -50475,5 +50475,455 @@
       "redirect_url": "/java/api/tests.integration.com.microsoft.azure.sdk.iot.transport._https_batch_message_i_t.UTF8().raw.page.js",
       "redirect_document_id": false
     },
+    {
+      "source_path": "docs-ref-services/ai-anomalydetector-readme-pre.md",
```

Comment:
Is this another manual update place for newly onboard services?
What's the purpose of having this?

Reply:
Let's have a quick call!